Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM ubuntu:19.10
+
+ENV BROWSER=/browser \
+    LC_ALL=en_US.UTF-8
+
+RUN echo '#!/bin/bash\n\
+\n\
+echo\n\
+echo "  $@"\n\
+echo\n\' > /browser && \
+    chmod +x /browser
+
+# runtime environment
+RUN apt-get update && \
+    apt-get install -y --no-install-suggests --no-install-recommends \
+      apt-utils ca-certificates gnupg2 locales curl python3 python3-distutils && \
+    echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && \
+    locale-gen && \
+    curl -L https://bootstrap.pypa.io/get-pip.py | python3 && \
+    pip3 install --no-cache-dir cython && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Intel MKL
```

Comment:
Do we really need it, right now (Kernel Math Library)?
The same with OpenMP?

Reply:
We are going to depend on pandas, pandas depends on numpy, and I custom-compile numpy to work faster with MKL.
So things like calculating stats should work faster.

There is a pypi package that does not force to do these things: https://pypi.org/project/intel-numpy/
But they don't provide a binary build for 3.7, unfortunately.

Reply:
I can remove these whistles and blows if everybody thinks it's not worth it...

Reply:
Since it's already here and it will make numpy go faster I'd just keep it.